src/api/searchCities.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import {
+    SearchCityTypes,
+    CIRCLE,
+    RADIUS,
+    COUNTRY_CODE,
+    LIMIT,
+    LANG
+} from '../utils/constans';
+
+import City from '../types/City/index'
+
+async function searchCities(search: string) {
+    const path = `${process.env.REACT_APP_HERE_API_BASE_URL}autocomplete`;
+    const cityParams = {
```

Comment:
np: in this case this object is config for get request (according to `axios.get` method specs), but not cityparams